### PR TITLE
adding rules spec tess with create device

### PR DESCRIPTION
**Pull request by kvish**
created a new device in my rules spec testing to work with

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@kvish__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

**Comment by handeadiguzel** [comment]
LGTM

File: apitests/spec/device_rules_spec.ts [hunk 1]
```diff
@@ -0,0 +1,137 @@
+const request = require('request').defaults({ json: true, baseUrl: 'https://localhost:44305/api/v1/devicerules' });
```
Comment (Inspiravetion) [inline apitests/spec/device_rules_spec.ts]: require once and set defaults in each beforeAll

**Comment by kvish** [comment]
updated request to be in beforeAll

File: apitests/spec/device_rules_spec.ts [hunk 2]
```diff
@@ -0,0 +1,136 @@
+
+describe('device rules api', () => {
+    //create a new device for use in tests
+    beforeAll(function() {
+        const request = require('request').defaults({ json: true, baseUrl: 'https://localhost:44305/api/v1/devicerules' });
```
Comment (Inspiravetion) [inline apitests/spec/device_rules_spec.ts]: I meant more like
```typescript
import * as request from 'request';

describe('', () => {
      beforeAll(function() {
             var req = request.defaults({...});
             //...
      });
})
```

**Comment by kvish** [comment]
got it, fixed it

**Comment by rraomsft** [comment]
LGTM

**Pull request merged**